frontend/src/components/ParticipantsListComponent/PrevPointTableBody.js

```diff
@@ -0,0 +1,66 @@
+// TODO:
+/**
+ * 1. Breakup into participant and visit child components
+ * 2. Break fields into smaller individual components for re-use in other forms
+ * 3. Break up state in MobX for field re-use
+ */
+
+/* eslint-disable indent */
+import React from "react"
+import PropTypes from "prop-types"
+import TableRow from "@material-ui/core/TableRow"
+import TableCell from "@material-ui/core/TableCell"
+import TableBody from "@material-ui/core/TableBody"
+import AssignmentIndIcon from "@material-ui/icons/AssignmentInd"
+import Fab from "@material-ui/core/Fab"
+import PrevPointCopy from "../../components/Typography/PrevPointCopy"
+import { Link } from "react-router-dom"
+
+const PrevPointTableBody = props => {
+  return (
+    <TableBody>
+      {props.participants.length < 1 ? (
+        <h5>Sorry no participants found</h5>
+      ) : null}
```

Comment:
You can just make the `null` statement here to be the `props.participants.map` that starts on the next line

Reply:
done